test/hide.md

```diff
@@ -277,6 +278,3 @@ _Validation:_
 
 _Appears in:_
 - [Underlying1](#underlying1)
-
-
-
```

Comment:
Tests fail because of this change.

Reply:
👍 Sorry, perhaps my editor's formatter removed those trailing new lines. I've just pushed a fix.